### DISCUSSION TITLE
Catalog registration page optimization.

### DISCUSSION THREAD

**Discussion by Aireed** [apache/amoro#1572] [General]
Currently,  the page is bind 
I think there are several points that need to be optimized now.

1. The registered information is strongly related to the Hadoop system.  
2. it's a little complicated for registration

**Comment by Aireed** [comment]
@baiyangtx @zhoujinsong

**Comment by xxubai** [comment]
+1

**Comment by baiyangtx** [comment]
Truely,  I think we should add a selection for storage system. and only upload site.xml if it's a hadoop system.

**Comment by majin1102** [comment]
+1
decoupled with Hadoop is always a good idea, shall we discuss some details?
